### PR TITLE
Fix watch launch problem in generated project

### PR DESCRIPTION
**Pull request by qyang-nj**
- [x] Built and ran in Xcode (`make project`)
- [x] Built and ran with Buck CLI (`make debug`)

Fix issue #97.

@bachand

### CODE REVIEW

File: App/WatchExtension/Info.plist [hunk 1]
```diff
@@ -32,9 +32,5 @@
 	</dict>
 	<key>WKExtensionDelegateClassName</key>
 	<string>$(PRODUCT_MODULE_NAME).ExtensionDelegate</string>
-	<key>UIRequiredDeviceCapabilities</key>
-	<array>
-		<string>arm64</string>
```
Comment (qyang-nj) [inline App/WatchExtension/Info.plist]: This is the culprit. Others are clean-up.

File: App/WatchApplication/Info.plist [hunk 2]
```diff
@@ -18,18 +18,16 @@
...
 	<key>CFBundleSupportedPlatforms</key>
 	<array>
 		<string>WatchSimulator</string>
 	</array>
 	<key>CFBundleVersion</key>
 	<string>1</string>
-	<key>DTPlatformName</key>
-	<string>watchsimulator</string>
-	<key>DTSDKName</key>
-	<string>watchsimulator2.0</string>
```
Comment (bachand) [inline App/WatchApplication/Info.plist]: Why are these not necessary?

File: App/WatchExtension/Info.plist (hunk 1 above)
Comment (bachand) [reply App/WatchExtension/Info.plist]: Huh. Since this is running on the phone it’s confusing why it would break...

Comment (qyang-nj) [reply App/WatchExtension/Info.plist]: Simulator isn't arm64.

File: App/WatchApplication/Info.plist (hunk 2 above)
Comment (qyang-nj) [reply App/WatchApplication/Info.plist]: Based on my limit observation, DT prefix indicates they are internal Xcode settings, which are usually inserted by Xcode automatically. Unlike other keys, those DT keys don't have any official doc to explain them.

File: App/WatchExtension/Info.plist (hunk 1 above)
Comment (bachand) [reply App/WatchExtension/Info.plist]: Got it. I recall we had to add this somewhere in order to upload the App Store. Hopefully it was just necessary for the main app and not this extension.

**Pull request merged**